src/main/java/seedu/address/logic/commands/TagEditCommand.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import java.util.List;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.Model;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Edits a tag in the tagList.
+ */
+public class TagEditCommand extends Command {
+
+    public static final String COMMAND_WORD = "edit";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits a tag to the taglist in the address book. "
+            + "Parameters: "
+            + PREFIX_TAG + "TAG"
+            + PREFIX_TAG + "TAG...\n"
+            + "Example: " + COMMAND_WORD + " "
```

Comment:
`tag `  should appear here too for the full command

Reply:
Made the changes. Thanks.